src/cpu/o3/BaseO3CPU.py

```diff
@@ -136,7 +136,11 @@ def support_take_over(cls):
     )
     renameToROBDelay = Param.Cycles(1, "Rename to reorder buffer delay")
     commitWidth = Param.Unsigned(8, "Commit width")
-    squashWidth = Param.Unsigned(8, "Squash width")
+    squashWidth = Param.Int(
```

Comment:
A few months ago, someone added "OptionalParam", which allowed us to have optional parameters. I think it'd be good to use them here instead of the utilization of negatives. I.e., if the squashWidth is not specified, it's assumed to be infinite, and all instructions are squashed within one cycle.

Reply:
Thanks for pointing this out! That's cool, didn't know about that.